example/tile.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const LifxClient = require('../lib/lifx').Client;
+
+const client = new LifxClient();
+
+const LOOPTIME = 1000;
+const TILE_LABEL = process.argv.length > 2 ? process.argv[process.argv.length - 1] : '*';
+
+/*
+ * This example should show on all tiles of
+ * your chain a random pattern. After all bits
+ * of the tiles are set with the random pattern,
+ * the example reads back all set values
+ * from the tiles just to test the read back function.
+ * I could not compare the set values out of the reason
+ * that the setvalues are usally a bit modified.
+ *
+ * EASY: You should see an updating Pattern every two second
+ */
+
+function getBits(light, idx, chain) {
+  if (idx >= chain.totalCount) {
+    console.log('All Bits get');
+    setTimeout(() => setBits(light, 0, chain), LOOPTIME);
+    return;
+  }
+  console.log('getBits', idx);
```

Comment:
Is this debugging info that needs to be removed? There are a few instances like these. Do we want to use `console.info` or `console.debug` instead? I always think of `console.log` statements as temporary for local development only.

Reply:
it an test  example programm, so why not 'log' that something is happens here?

Reply:
I think these are fine, they indicate at least what part of the code is currently executing. They could of course be more informative as I'm suspecting the API might be quite difficult to graps for many.